index.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="author" content="Mirela Pitt" />
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v6.0.0/css/all.css"
+    integrity="sha384-3B6NwesSXE7YJlcLI9RpRqGf2p/EgVH8BgoKTaUrmKNDkHPStTQ3EyoYjCGXaOTS" crossorigin="anonymous" />
+  <title>Fashion on me</title>
+</head>
+
+<body>
+  <header>
+    <div class="bg"></div>
+    <!--Desktop Menu-->
+    <div class="before-nav">
+      <ul class="before-nav-content">
+        <li><a><i class="fa-brands fa-facebook-f"></i></a></li>
+        <li><a><i class="fa-brands fa-twitter"></i></a></li>
+        <li><a>English</a></li>
+        <li><a>My page</a></li>
+        <li><a>logout</a></li>
+      </ul>
+    </div>
+
+    <!--Mobile Menu-->
+    <nav>
+      <div class="logo">
+        <h1><a href="#home">DC Fashion Show</a></h1>
+      </div>
+
+      <div class="burger" id="burger">
+        <div class="line"></div>
+        <div class="line"></div>
+        <div class="line"></div>
+      </div>
+
+      <div class="nav-links" id="navLinks">
+        <ul class="desktop-navlinks" id="navItems">
+          <li id="closeMenu"><i class="fa-solid fa-xmark" id="closeMenuIcon"></i></li>
+          <li class="info"><a href="about.html">about</a></li>
+          <li class="info"><a href="#about">program</a></li>
+          <li class="info"><a href="#contact">join</a></li>
+          <li class="info"><a href="#contact">sponsor</a></li>
+          <li class="info"><a href="#contact">news</a></li>
+          <li class="info"><button type="button" class="nav-button">cc company</button></li>
+        </ul>
+      </div>
```

Comment:
- [ ] Kindly add more padding-top and padding-bottom to your `nav ` :sunglasses::nerd_face:
- [ ] You can consider applying this to the About Page since they have the same header  🎨 🖌️  I know you can do it :muscle: 😄 



- [Cindy Shin](https://www.behance.net/adagio07)[Cindy Shin](https://www.behance.net/adagio07) design:

![header41](https://user-images.githubusercontent.com/79058364/170614501-2183a1cb-71cb-4eab-b3ff-87b9c447096a.png)


- Your Design:

![header42](https://user-images.githubusercontent.com/79058364/170614399-37cffea8-4e91-431b-b3a9-304608fcb9f4.png)